\documentclass[12pt]{letter}
%DIF LATEXDIFF DIFFERENCE FILE
%DIF DEL reply_to_referees_2.tex       Sat Nov 30 20:57:35 2019
%DIF ADD reply_to_referees_2_dbs.tex   Sun Dec  8 09:44:47 2019

 \usepackage[britdate]{SU-letter}
\usepackage{times}
\usepackage{letterbib}
\usepackage{geometry}
\usepackage[numbers,sort&compress]{natbib}
\usepackage{graphicx}
\geometry{a4paper}
\usepackage[T1]{fontenc}
\usepackage[utf8]{inputenc}
\usepackage{authblk}
\usepackage[running]{lineno}
\usepackage{amsmath,amsfonts,amssymb}
% \usepackage[margin=10pt,font=small,labelfont=bf]{caption}

%\usepackage{natbib}
% \bibpunct[; ]{(}{)}{;}{a}{,}{;}

\newenvironment{refquote}{\bigskip \begin{it}}{\end{it}\smallskip}

\newenvironment{figure}{}


\position{Postdoctoral fellow}
\department{Department of Ecology, Evolution, and Plant Sciences}
\location{Private Bag 4800}
%DIF 28c28
%DIF < \telephone{+64 3 364 2729}
%DIF -------
% \telephone{+64 3 364 2729} %DIF > 
%DIF -------
\fax{+64 3 364 2590}
\email{alyssa.cirtwill@gmail.com}
\url{http://cirtwill.github.io}
\name{Alyssa R. Cirtwill}




\newcommand{\mytitle}{\emph{At a global scale, conservation of pollinators and herbivores between related plants varies widely across communities and between plant families.}}
\newcommand{\myjournal}{\emph{New Phytologist}}
 %DIF > 
%DIF PREAMBLE EXTENSION ADDED BY LATEXDIFF
%DIF UNDERLINE PREAMBLE %DIF PREAMBLE
\RequirePackage[normalem]{ulem} %DIF PREAMBLE
\RequirePackage{color}\definecolor{RED}{rgb}{1,0,0}\definecolor{BLUE}{rgb}{0,0,1} %DIF PREAMBLE
\providecommand{\DIFadd}[1]{{\protect\color{blue}\uwave{#1}}} %DIF PREAMBLE
\providecommand{\DIFdel}[1]{{\protect\color{red}\sout{#1}}}                      %DIF PREAMBLE
%DIF SAFE PREAMBLE %DIF PREAMBLE
\providecommand{\DIFaddbegin}{} %DIF PREAMBLE
\providecommand{\DIFaddend}{} %DIF PREAMBLE
\providecommand{\DIFdelbegin}{} %DIF PREAMBLE
\providecommand{\DIFdelend}{} %DIF PREAMBLE
%DIF FLOATSAFE PREAMBLE %DIF PREAMBLE
\providecommand{\DIFaddFL}[1]{\DIFadd{#1}} %DIF PREAMBLE
\providecommand{\DIFdelFL}[1]{\DIFdel{#1}} %DIF PREAMBLE
\providecommand{\DIFaddbeginFL}{} %DIF PREAMBLE
\providecommand{\DIFaddendFL}{} %DIF PREAMBLE
\providecommand{\DIFdelbeginFL}{} %DIF PREAMBLE
\providecommand{\DIFdelendFL}{} %DIF PREAMBLE
\newcommand{\DIFscaledelfig}{0.5}
%DIF HIGHLIGHTGRAPHICS PREAMBLE %DIF PREAMBLE
\RequirePackage{settobox} %DIF PREAMBLE
\RequirePackage{letltxmacro} %DIF PREAMBLE
\newsavebox{\DIFdelgraphicsbox} %DIF PREAMBLE
\newlength{\DIFdelgraphicswidth} %DIF PREAMBLE
\newlength{\DIFdelgraphicsheight} %DIF PREAMBLE
% store original definition of \includegraphics %DIF PREAMBLE
\LetLtxMacro{\DIFOincludegraphics}{\includegraphics} %DIF PREAMBLE
\newcommand{\DIFaddincludegraphics}[2][]{{\color{blue}\fbox{\DIFOincludegraphics[#1]{#2}}}} %DIF PREAMBLE
\newcommand{\DIFdelincludegraphics}[2][]{% %DIF PREAMBLE
\sbox{\DIFdelgraphicsbox}{\DIFOincludegraphics[#1]{#2}}% %DIF PREAMBLE
\settoboxwidth{\DIFdelgraphicswidth}{\DIFdelgraphicsbox} %DIF PREAMBLE
\settoboxtotalheight{\DIFdelgraphicsheight}{\DIFdelgraphicsbox} %DIF PREAMBLE
\scalebox{\DIFscaledelfig}{% %DIF PREAMBLE
\parbox[b]{\DIFdelgraphicswidth}{\usebox{\DIFdelgraphicsbox}\\[-\baselineskip] \rule{\DIFdelgraphicswidth}{0em}}\llap{\resizebox{\DIFdelgraphicswidth}{\DIFdelgraphicsheight}{% %DIF PREAMBLE
\setlength{\unitlength}{\DIFdelgraphicswidth}% %DIF PREAMBLE
\begin{picture}(1,1)% %DIF PREAMBLE
\thicklines\linethickness{2pt} %DIF PREAMBLE
{\color[rgb]{1,0,0}\put(0,0){\framebox(1,1){}}}% %DIF PREAMBLE
{\color[rgb]{1,0,0}\put(0,0){\line( 1,1){1}}}% %DIF PREAMBLE
{\color[rgb]{1,0,0}\put(0,1){\line(1,-1){1}}}% %DIF PREAMBLE
\end{picture}% %DIF PREAMBLE
}\hspace*{3pt}}} %DIF PREAMBLE
} %DIF PREAMBLE
\LetLtxMacro{\DIFOaddbegin}{\DIFaddbegin} %DIF PREAMBLE
\LetLtxMacro{\DIFOaddend}{\DIFaddend} %DIF PREAMBLE
\LetLtxMacro{\DIFOdelbegin}{\DIFdelbegin} %DIF PREAMBLE
\LetLtxMacro{\DIFOdelend}{\DIFdelend} %DIF PREAMBLE
\DeclareRobustCommand{\DIFaddbegin}{\DIFOaddbegin \let\includegraphics\DIFaddincludegraphics} %DIF PREAMBLE
\DeclareRobustCommand{\DIFaddend}{\DIFOaddend \let\includegraphics\DIFOincludegraphics} %DIF PREAMBLE
\DeclareRobustCommand{\DIFdelbegin}{\DIFOdelbegin \let\includegraphics\DIFdelincludegraphics} %DIF PREAMBLE
\DeclareRobustCommand{\DIFdelend}{\DIFOaddend \let\includegraphics\DIFOincludegraphics} %DIF PREAMBLE
\LetLtxMacro{\DIFOaddbeginFL}{\DIFaddbeginFL} %DIF PREAMBLE
\LetLtxMacro{\DIFOaddendFL}{\DIFaddendFL} %DIF PREAMBLE
\LetLtxMacro{\DIFOdelbeginFL}{\DIFdelbeginFL} %DIF PREAMBLE
\LetLtxMacro{\DIFOdelendFL}{\DIFdelendFL} %DIF PREAMBLE
\DeclareRobustCommand{\DIFaddbeginFL}{\DIFOaddbeginFL \let\includegraphics\DIFaddincludegraphics} %DIF PREAMBLE
\DeclareRobustCommand{\DIFaddendFL}{\DIFOaddendFL \let\includegraphics\DIFOincludegraphics} %DIF PREAMBLE
\DeclareRobustCommand{\DIFdelbeginFL}{\DIFOdelbeginFL \let\includegraphics\DIFdelincludegraphics} %DIF PREAMBLE
\DeclareRobustCommand{\DIFdelendFL}{\DIFOaddendFL \let\includegraphics\DIFOincludegraphics} %DIF PREAMBLE
%DIF END PREAMBLE EXTENSION ADDED BY LATEXDIFF

\begin{document}

\begin{letter}{\bf Dr. Jana Vamosi\\
               Editor\\
               Proceedings of the Royal Society B\\
               6-9 Carlton House Terrace\\
               London, UK\\
               SW1Y 5AG\\
                }

\opening{Dear Dr. Vamosi:}

	We are pleased to submit a revised version of our manuscript, NPH-MS-2019-30803. In response to a suggestion from Referee 3, we have re-titled it "". We have also taken care to revise our methods section in response to the many helpful suggestions from Referee 4, and hope that our approach is now clearer. Finally, we have expanded the discussion to include some additional concepts suggested by Referees 2 and 4, and a possible application of our research suggested by Referee 3. 


	Given the difference of opinion between Referees 1, 3, and 4 and Referee 2, we have prioritised adjusting the manuscript according to the majority view. We must also admit to having some difficulty parsing a number of Referee 2's comments in order to find what \DIFdelbegin \DIFdel{is }\DIFdelend \DIFaddbegin \DIFadd{precisely was }\DIFaddend being asked. This may be partly because this review came from several people - the Referee refers to themselves as 'we' several times, and the writing appears to be more than usually variable within their comments. These difficulties mean that we may not have followed as many of Referee 2's suggestions as we have for the other Referees. Nevertheless, we do appreciate their time and effort in offering such extensive feedback and have taken care to address each of Referee 2's comments \DIFaddbegin \DIFadd{to the best of our abilities }\DIFaddend in the hope of at least clarifying our perspective.


	We appreciate all of the Referees' comments and are confident that our manuscript is all the stronger for responding to them.


\closing{Regards,}


\end{letter}

\newpage

\setcounter{page}{1}

% NPH-MS-2019-30803 At a global scale, conservation of pollinators and herbivores between related plants varies widely across communities and between plant families.
% by Cirtwill, Alyssa; Dalla Riva, Giulio; Baker, Nick; Ohlsson, Mikael; Norström, Isabelle; Wohlfarth, Inger-Marie; Thia, Josh; Stouffer, Daniel

% Dear Dr Cirtwill

% I write you in regard to your submitted manuscript, NPH-MS-2019-30803 "At a global scale, conservation of pollinators and herbivores between related plants varies widely across communities and between plant families.”, which has now been reviewed. We found four reviewers (some original reviewers and some new) and they all generally enjoyed your revised manuscript. However, they also provided suggestions for some expanding certain sections and providing additional clarification. In particular, Reviewer \#2 and \#4 provide some detailed suggestions, especially with regard to keeping the aims and logic consistent, that will help strengthen the manuscript.

% Although resolving these issues will require some restructuring, I think you should be able to handle these suggestions through revision (see below) and, ultimately, this manuscript will make a nice contribution to the literature.

% Please carefully revise your manuscript and resubmit your revised version within 4 weeks. Please also include a summary of how you have incorporated the suggestions of the reviewers.

% Thank you for submitting your work to New Phytologist.

% Sincerely,

% Jana Vamosi

% Editor, New Phytologist


% Decision: Accept subject to revision

\clearpage

{\Large \bf Reply to Referee 1}

	\begin{refquote}
	Having read the manuscript and the response to previous reviewers’ comments, I do not see any further changes that need to be made. Any concerns I would have had, have been addressed by the latest round of revisions. I did not even spot any typos!
	\end{refquote}


	\textbf{R:} We thank the referee for taking the time to read our manuscript and for the compliments to our revision and proof-reading.


\clearpage

{\Large \bf Reply to Referee 2} 


	\begin{refquote}
	General comments\\

	The manuscript proposes to evaluate how phylogeny modulates species interaction patterns in plant-pollinator and a subset of plant-herbivory networks. In this regard, it explores an extremely relevant question for ecologists. However, it still needs work on the logic of the introduction section and methodological decisions need to be revised in order to achieve more confident results. The Introduction lacks focus and presents unclear aims, which leads to an unfocused discussion of the results, being one of the main weaknesses of the manuscript. Also, there is a lack of logical consistency between the abstract (methodologically-oriented) and the introduction and discussion sections (both biologically-oriented). On the other hand, we do not understand why authors do not incorporate ideas of species evolution and coevolution in a network context if you are trying to understand how phylogenetic constraints may modulate species interaction patterns in networks. Considering such exciting ideas may help to understand some of your results and may advance our understanding of  the importance evolutionary processes in determining species interaction patterns in communities.
	Another main point that you should revise across your manuscript is that you confound phylogenetic patterns with taxonomic patterns, for instance when thinking about congeneric species as closely related species (which is not necessarily true in several cases).

	\smallskip

	In what follows we share with authors our specific comments expecting that they will help to re-think the manuscript and get a stronger contribution.

	\end{refquote}


	The Referee \DIFdelbegin \DIFdel{(s?) }\DIFdelend seems to view our basic question and analyses positively, but is rather \DIFdelbegin \DIFdel{bluntly }\DIFdelend critical of our writing in terms of logical, clarity, and style.
	\DIFdelbegin \DIFdel{Some of these criticisms seem to reflect a rather hasty reading of the manuscript, as there }\DIFdelend %DIF > Some of these criticisms seem to reflect a rather hasty reading of the manuscript, as
	\DIFaddbegin \DIFadd{There }\DIFaddend are points where the Referee asks for analyses or ideas to be included which \DIFdelbegin \DIFdel{already appear }\DIFdelend \DIFaddbegin \DIFadd{we felt already appeared }\DIFaddend in the text.
	In these cases, we point to \DIFdelbegin \DIFdel{existing evidence in the manuscript }\DIFdelend \DIFaddbegin \DIFadd{the areas in question }\DIFaddend and have tried to make these connections more \DIFdelbegin \DIFdel{obvious}\DIFdelend \DIFaddbegin \DIFadd{apparent}\DIFaddend .
	In a few \DIFaddbegin \DIFadd{further }\DIFaddend cases, we \DIFdelbegin \DIFdel{struggled to understand the nature }\DIFdelend \DIFaddbegin \DIFadd{were not fully able to identify the core }\DIFaddend of the Referee's objections. We have attempted to address each of the Referee's comments in detail below (preceded by \textbf{R:}). Wherever possible, we have clarified our text in response to the Referee's comments. In particular, we now define community composition more explicitly, have revised the discussion to be more explicit about whether we are referring to overall or within-network trends, and have removed specific phrases that the Referee \DIFdelbegin \DIFdel{finds }\DIFdelend \DIFaddbegin \DIFadd{found }\DIFaddend objectionable \emph{prima facie}. We hope that these changes and our response will clarify the focus and scope of our analyses.


	1. General criticism of the language of the text

		\begin{refquote}

		Finally, the text needs English revision by the authors as we feel that some ideas are not clearly stated because of the way they are presented to readers. 

		\end{refquote}


		\textbf{R:} 
		\DIFdelbegin \DIFdel{As Referees 1 and 3 note, our English is scrupulously correct. Of course technical correctness does not necessarily imply clarity of expression, and we apologise if our choice of vocabulary has obfuscated some of the nuances of the manuscript}\DIFdelend %DIF > As Referees 1 and 3 note, our English is scrupulously correct. Of course technical correctness does not necessarily imply clarity of expression, and we apologise if our choice of vocabulary has obfuscated some of the nuances of the manuscript. We have endeavoured to clarify the manuscript as the Referee suggests.
		\DIFaddbegin \DIFadd{We are sorry the Referee found the text less clear than we would have desired}\DIFaddend . We have \DIFdelbegin \DIFdel{endeavoured to clarify the manuscript as the Referee suggests}\DIFdelend \DIFaddbegin \DIFadd{gone through the latest version very carefully with a focus on the clarity of the text}\DIFaddend .



	2. Title lacks clarity

		\begin{refquote}
		Specific comments\\
		Title\\
		Your title still lacks clarity and does not reflect the main message you state at the beginning of your discussion (in the first sentence of it you state that you found broad support for the hypothesis that more closely related plants tend to have more niche overlap). We feel that this reflects lack of logical consistency across your manuscript, so re-thinking it should help to define your title.
		\end{refquote}

		\textbf{R:} We have revised the title following a more detailed suggestion by Referee 3 and hope that the new title \DIFdelbegin \DIFdel{no longer implies logical inconsistency}\DIFdelend \DIFaddbegin \DIFadd{is a better reflection of the manuscript as a whole}\DIFaddend .


	3. Summary too methodologically-oriented in general

		\begin{refquote}
			Summary\\
			General comments\\
			Your summary is too methodologically-oriented. In fact, when we were invited to be reviewers of your manuscript and received your abstract we thought that your paper represented a methodological contribution and it is not. After reading the comments that former reviewers sent you, your abstract seems to correspond to an older version of your manuscript. The thing is that by presenting an abstract that is methodologically-oriented when in fact you are not innovating in this sense, and having an introduction and a discussion that lack such methodological issue and are clearly more biologically-oriented can be read as that your work lacks focus.
		\end{refquote}

		\begin{refquote}
			Your 2nd paragraph is too technical: which is the biological meaning of it? Why are you going to compare pollination and herbivory networks? Why your datasets are so unbalanced? Why are you considering a wide range of pollination networks and a narrow one for herbivory networks? Why you consider shared and unshared partners? Your first point in the abstract doesn't introduce a methodological problem to be solve. (...) "and whether this relationship varies with the composition of the plant community"? What does it mean?
		\end{refquote}

		\textbf{R:} We regret that the Referee \DIFdelbegin \DIFdel{finds }\DIFdelend \DIFaddbegin \DIFadd{found }\DIFaddend our summary too methodologically-orientated\DIFdelbegin \DIFdel{, but }\DIFdelend \DIFaddbegin \DIFadd{. We also }\DIFaddend note that Referee 4 has requested that we include further methodological details in the summary. \DIFdelbegin \DIFdel{Moreover, in a later comment this Referee also requests more methodological details. }\DIFdelend To balance these \DIFdelbegin \DIFdel{conflicting comments}\DIFdelend \DIFaddbegin \DIFadd{out}\DIFaddend , we no longer emphasise the methodological novelty of our study but have added the greater detail requested. \DIFdelbegin \DIFdel{As the other three Referees for this revision did not find the summary overly methodological in its original form, we are confident that the revised summary does not indicate that we lack focus. }\DIFdelend We also note\DIFaddbegin \DIFadd{, however, }\DIFaddend that the extremely tight word limit for the summary does not allow us to explain why we have unbalanced numbers of pollination and herbivory networks. This information is contained in the \DIFdelbegin \DIFdel{methods}\DIFdelend \DIFaddbegin \DIFadd{Methods}\DIFaddend . 


	4. Mis-understanding of purpose and scope of within-family analyses 

		\begin{refquote}
		Your abstract also evidences your confusion with phylogenetic issues and taxonomic ones. For instance, your hypothesis is that more closely related plants will share more pollinators and herbivores than less closely related. Then, why searching for different relationships within families? This decision is completely arbitrary is one of the weakest point of the manuscript in general.
		\end{refquote}


		\textbf{R:} The Referee is correct about our hypothesis but \DIFdelbegin \DIFdel{appears to have assumed }\DIFdelend \DIFaddbegin \DIFadd{it appears that our earlier version lead them to believe }\DIFaddend that we are \emph{only} searching for conservation of niche overlap within families. In fact, point 2 of the summary lists three separate analyses: testing for relationships between phylogenetic distance and partner overlap in \emph{all pairs of plants in a network, regardless of family}, testing whether the slopes of these whole-network relationships are related to the composition of the plant community in each network, and, as a supplement to these analyses, testing whether the slopes of these relationships differ if we consider only pairs of plants within the same family. We now number these three different analyses to avoid any further confusion. 


		In this context, checking for different relationships within families is similar to checking for different relationships between networks - an overall trend can mask a great deal of variability, and we wanted to make sure this variability was identified \DIFaddbegin \DIFadd{as best possible}\DIFaddend . We chose families as the taxonomic level for further analysis because many plant families are quite diverse, both in terms of numbers of species and in terms of phenotypes/interaction partners (see e.g., Wilson on plants and insect/bird pollinators). Within a single family, plants can show evidence of adaptive radiations, heterogeneous selection, both, or neither. The Referee themself points out that some families can show both bird and bee pollination syndromes (see response \#8), and we can also point to papers such as Bernhardt 2000 and Johnson \& Jurgens 2010 which address trait lability, convergent evolution, and adaptive radiation. These different possibilities mean that it is not clear whether all families will show phylogenetic conservation of interaction partners. 
		\DIFdelbegin \DIFdel{We have }\DIFdelend \DIFaddbegin \DIFadd{As we seem to have failed in making this point clearly in the earlier version, we have }\DIFaddend added a summary of this explanation to the methods. 


		Lines 252-257:

		\begin{quotation}

			Finally, we compared the breakdown of niche overlap in different plant families.
		    Within-family genetic and trait diversity can be high due to adaptive radiations, heterogeneous selection, and other influences on different species. 
		    Plant families offer a reasonable balance between collecting enough species to identify meaningful trends and maintaining a tractable number of analyses. 
		    They are therefore the best taxonomic level to investigate phylogenetic conservation in more detail across our large dataset.

		\end{quotation}


		As we are \DIFdelbegin \emph{\DIFdel{not}} %DIFAUXCMD
\DIFdelend \DIFaddbegin \DIFadd{not }\DIFaddend searching for differences \DIFdelbegin \DIFdel{between }\DIFdelend \DIFaddbegin \emph{\DIFadd{between}} \DIFaddend communities at the family level, but rather at all levels within the angiosperms, a number of the Referee's comments \DIFdelbegin \DIFdel{do not apply. We }\DIFdelend \DIFaddbegin \DIFadd{did not seem to directly apply. I their brevity, it is possible that we were unable to capture the Referee's principle argument, so we }\DIFaddend list these below \DIFaddbegin \DIFadd{in case they prefer greater clarification going forward}\DIFaddend . 


		\begin{refquote}
		3rd paragraph:  which is your biological argument to consider families as the taxonomic level to search for differences across communities?
		\end{refquote}


		\begin{refquote}
		Aims
		Why considering the phylogenetic signal at the plant families level? There is no introduction for this aim. Why focusing at such taxonomic level?
		\end{refquote}


		\begin{refquote}
			Line 318. Within-family conservation of niche overlap. Your result is almost expected so we think is important to justify biologically why you decided to look at the family level.
		\end{refquote}


	5. \DIFdelbegin \DIFdel{Very confusing comment complaining }\DIFdelend \DIFaddbegin \DIFadd{Comments }\DIFaddend about summary point 4

		\begin{refquote}
		4st paragraph "Considering factors affecting the dominant plant families within
		a community may be the key to understanding the distribution of interactions." So, relationships are more functional? Why focusing on plant families? we are not sure the logic of your thoughts can be followed (can the reader reach this conclusion? is it ok? we mean, if there are other eco-evolutionary processes determining species interactions why focusing in the dominant plant families would be informative? I feel the way this last paragraph is written isn't clear enough and is your main paragraph.
		\end{refquote}


		\textbf{R:} Apart from the Referee's objection to plant families as units for detailed analysis (see \DIFdelbegin \DIFdel{respons }\DIFdelend \DIFaddbegin \DIFadd{response }\DIFaddend \#4), this comment seems to demonstrate uncertainty about how we defined the dominant plant families in a community. Note that we do not mean to imply any functional argument here, as our manuscript does not address functionality. Rather, by 'dominant', we mean 'most species-rich in the focal community'. In order to avoid future confusion, we now replace 'dominant families' with 'species-rich groups'. We hope that this makes it clear that we are advocating \DIFdelbegin \DIFdel{researcher }\DIFdelend \DIFaddbegin \DIFadd{researchers }\DIFaddend take a finer-scale perspective on phylogenetic signal in ecological networks.


		\begin{quotation}
		    \item The variety of relationships between phylogenetic distance and partner overlap in different plant families likely reflects a comparable variety of ecological and evolutionary processes. Considering factors affecting particular species-rich groups within a community may be the key to understanding the distribution of interactions.
		\end{quotation}


	6. Complaint about keywords absent from the abstract

		\begin{refquote}
		Keywords\\
		Syndrome:  Why you do not mention such theory in your abstract?
		\end{refquote}


		\textbf{R:} The abstract/summary is under strict word limits, and we did not feel that inserting each keyword without a clear purpose could justify the elimination of other material.
		\DIFdelbegin \DIFdel{Given the Referee's confusion regarding our analyses, we are more convinced than ever that our decision to omit some keywords in favour of spending more words explaining our methodology was correct. }\DIFdelend %DIF > Given the Referee's confusion regarding our analyses, we are more convinced than ever that our decision to omit some keywords in favour of spending more words explaining our methodology was correct.
		As we do not directly test pollination or defensive syndromes, we have removed these keywords from the list. In response to the Referee's further objections to the syndrome literature, below, we have also removed these terms from the introduction.


	7. \DIFdelbegin \DIFdel{Mistake about the }\DIFdelend \DIFaddbegin \DIFadd{The }\DIFaddend focus of the introduction

		\begin{refquote}
		Introduction\\
		General. You focus on the compromise between pollination and herbivory interactions but then you do not use this approach in your analyses.
		\end{refquote}


		\textbf{R:} Potential tradeoffs between encouraging interactions with pollinators and discouraging herbivores are just one of the possible reasons why networks may not display the expected relationship between relatedness and niche overlap. Note that we also propose several others in the same paragraph where trade-offs appear, including convergent evolution or selection to avoid pollen dilution. We therefore disagree with the Referee's perception that \DIFdelbegin \DIFdel{we focus on }\DIFdelend \DIFaddbegin \DIFadd{out primary focus is the }\DIFaddend compromise between pollination and herbivory.
		\DIFdelbegin \DIFdel{It is beyond the scope of this manuscript to test all possible reasons why closely-related plants might not share interaction partners. }\DIFdelend %DIF >  It is beyond the scope of this manuscript to test all possible reasons why closely-related plants might not share interaction partners.
		This material is intended solely as background for readers who may not be familiar with arguments against the phylogenetic conservation of interaction partners. \DIFaddbegin \DIFadd{We have reviewed the text in question to reassure ourselves that this is the tenor of our discussion.
}\DIFaddend 


	8. Disagreement with any link between syndromes and conservation of interaction partners

		\begin{refquote}
			1st paragraph: We suggest this paragraph needs to be more functionally-oriented than phylogenetically-oriented. Syndromes may respond to the convergence of differently related organisms to a similar phenotype to attract the same pollinators. That is why plant syndromes can be more related with higher levels of taxonomic classification (orders) or to functionally defined groups like short and long-tongued bees, and not necessarily with sharing exactly the same pollinator species. Are you aware of articles showing that plant syndromes are phylogenetically conserved? We can think about many genera in which closely related plants show the same syndrome but bee or hummingbird syndromes are found within the family, so why should we expect that phylogeny has a strong signal? Or that more closely related plants will show more similar pollinators?
		\end{refquote}

		\begin{refquote}
			2nd paragraph: your reasoning is not clear for us. Syndromes imply the convergence of phenotypic traits into one common set of traits because of their functionality in increasing fitness (because of the selection pressures imposed by functional groups of pollinators that can or cannot be closely related). Even if traits are heritable, if many unrelated groups converge into one functional group of interaction partners, why you may expect more similarity among closely related species? On the other hand, can we expect modular structures based on your reasoning? Then, how you explain the high incidence of nested structures in both mutualistic and antagonistic networks? In fact, what has been proposed for coevolution in mutualistic networks is the convergence of species traits to those of the species forming the core (see Guimaraes et al 2017, Nuismer et al. 2018). In antagonistic networks your line of reasoning may apply, but this will clearly depend on the intimacy of your network as most herbivory networks tend to show nested structures too. Your reasoning is based on pairwise coevolution and does not incorporate ideas of evolution and coevolution in networks, which may completely alter your predictions about niche overlap.
		\end{refquote}

		\begin{refquote}
			Lines 65-65. You wrote: Unrelated plants might also converge upon similar phenotypes, attracting a particularly efficient or abundant pollinator (Ollerton, 1996; Wilson et al., 2007; Ollerton et al., 2009; Ibanez et al., 2016). This is exactly our point: pollination syndromes may reflect trait convergence among unrelated species.
		\end{refquote}


		\textbf{R:} As the Referee notes, we do address the possibility of convergence in our introduction. We do not make any claim as to whether the putative pollination syndromes actually reflect plants with common sets of pollinators (the evidence we are aware of is rather mixed). Since the Referee appears to object to this idea (which we \DIFaddbegin \DIFadd{nevertheless }\DIFaddend note is quite common in the literature surrounding phylogenetic conservation of interactions), and because syndromes do not form a major part of our analysis, we have removed it as a keyword and rephrased our introduction so that it now no longer references pollination syndromes. Hopefully this will help readers to \DIFdelbegin \DIFdel{take }\DIFdelend \DIFaddbegin \DIFadd{evaluate }\DIFaddend our manuscript on its own merits rather than those of the syndrome concept.


		Lines 11-14:

		\begin{quotation}

			Plants with different defences 
		  	(e.g., thorns vs. chemical defences) may deter different groups of 
		  	herbivores~\citep{Ehrlich1964,Johnson2014}, and pollinators with similar traits are often expected to attract similar sets of pollinators~\citep{Waser1996,Fenster2004,Ollerton2009}.

		\end{quotation}


	9. Request to include interaction intimacy

		\begin{refquote}
		Lines 35-39. Maybe you should use the idea of interaction intimacy, since it clearly will determine the level of selective pressure landscape perceived by a given species?
		\end{refquote}


		\textbf{R:} We have no information on interaction strengths or intimacies, and we note that the interactions we consider are generally low-intimacy. We therefore would not expect intimacy to strongly affect these networks. However, we are happy to include the idea as yet another reason why plant communities may or may not show conservation of interactions. We have added a brief description of intimacy as follows:


		Lines 29-37:

		\begin{quotation}

			In mutualistic networks, animals often show a stronger phylogenetic signal in their partners than do plants~\citep{Rezende2007a,Chamberlain2014,Rohr2014,Vamosi2014,Lind2015,Fontaine2015} (but see~\citet{Rafferty2013} for a counterexample). In antagonistic networks, however, actively-foraging consumers tend to show less phylogenetic signal than their prey~\citep{Ives2006,Cagnolo2011,Naisbit2011,Fontaine2015}. In part, this may be related to different degrees of interaction intimacy (dependence of one partner on another), which appears to contribute to network structure in mutualistic, but not antagonistic, networks~\citep{Guimaraes2007,Ponisio2017}. 

		\end{quotation}


		In addition to the above, we have added another brief discussion of intimacy in the discussion (see response \#28)


	10. Criticism of dataset \DIFdelbegin \DIFdel{changes required by previous reviewers
}\DIFdelend %DIF > changes required by previous reviewers


		\begin{refquote}
		Methods\\
		Dataset\\
		Several networks can be reviewed in the literature or are available in different websites. Why authors decide not to consider several plant-herbivore networks but have no filter to separate plant-pollinator networks? The reason of why focusing in plant-pollinator and plant-herbivory networks is not clear for us as well why authors choose to select just one kind of herbivory interaction and as a consequence have a highly unbalanced design. Such decision seems to be crucial as it may modulate the results and conclusions they achieve. By eliminating the herbivory interactions that are more phylogenetically constrained their result that pollination interactions may be more conserved than herbivory interactions seems to be a by-product of such decision.
		\end{refquote}


		\textbf{R:} We included all herbivory networks we could find when we began this project, and only restricted our selection of herbivory networks in response to a previous round of review. Without more details \DIFaddbegin \DIFadd{regarding }\DIFaddend why (or if) the Referee believes that the herbivory networks we removed should be included, we have opted to continue following that earlier feedback. We would have appreciated specific tips \DIFdelbegin \DIFdel{for }\DIFdelend \DIFaddbegin \DIFadd{pointing us toward }\DIFaddend additional herbivory networks, but \DIFdelbegin \DIFdel{since the Referee does not provide any specifics we do not feel that  conducting an extensive }\DIFdelend \DIFaddbegin \DIFadd{lacking this information we concluded that  conducting a further }\DIFaddend search for additional networks and repeating our analyses \DIFdelbegin \DIFdel{is viable }\DIFdelend \DIFaddbegin \DIFadd{was un-viable }\DIFaddend within the timeframe given to respond to this review.


	11. Request to change "insects consuming leaves" to "chewing insects"

		\begin{refquote}
			Lines 93 to 96. We think that sap-sucking, leaf mining, and galling insects are also insect consuming leaves. Maybe you could change "insect consuming leaves" by "chewing insects"
		\end{refquote}

		\textbf{R:} The term "insects consuming leaves" was recommended by an earlier reviewer, but we agree that leaf mining also constitutes leaf consumption (this is not necessarily the case for sap-sucking and galling, which is less leaf-specific). We have changed these lines as suggested.


	12. Recommendation to continue using Jaccard dissimilarity, binomial regression

		\begin{refquote}
			Measuring niche overlap\\
			Why you used absolute numbers to measure shared and unshared pollinators and not the Jaccard index? Or, instead, why you did not use a binomial regression considering the absolute number of the whole set of pollinators that can be shared ? You should use generalized linear models with a binomial distribution considering the total number of pollinators and shared pollinators to give more weight to those species sharing more pollinators in a larger sample (generalists s for instance). It is not clear for us if your analysis considers the full number of species over which you obtain the percentage of shared species.
		\end{refquote}

		\textbf{R:} We \DIFdelbegin \DIFdel{did }\DIFdelend \DIFaddbegin \DIFadd{apologize for any confusion here, as we did in fact }\DIFaddend use a Jaccard index, as stated in the first  sentence under the sub-heading \textbf{Calculating niche overlap between communities} in the previous revision. This line read as follows (emphasis added): 
		\begin{quotation}
		We calculated niche overlap for each pair of species within a community \emph{using a Jaccard index} to describe the number of shared interaction partners, augmented with the number of interaction partners which were not shared.
		\end{quotation}

		The Jaccard index is calculated using the number of shared and un-shared interaction partners for a pair of plants, as stated in equation 1. We also did use a generalised linear model with binomial error distribution, as stated in the \textbf{Statistical analysis} section. We chose this methodology for exactly the reasons the Referee suggests, to give more weight to generalists sharing many interaction partners. This is stated in lines 138-141 and lines 148-154 (quoted below). Note that we have revised these lines somewhat in response to a suggestion from Referee 4, so they do differ somewhat from the previous version. We are glad that the Referee agrees with our choice in methodology.

		\begin{quotation}

			Together, $M_{ij}$ and $U_{ij}$ give a Jaccard index ($J_{ij}$) describing 
			the proportion of shared interactions. $J_{ij}$ is defined: 

			\begin{equation}
			J_{ij} = \frac{M_{ij}}{U_{ij}+M_{ij}} ,
			\end{equation}
			%
			where $M_{ij}$ is the set of \emph{mutual} (shared) interaction partners and $U_{ij}$ the set of unshared interaction partners for plants $i$ and $j$.

		\end{quotation}

		\smallskip

		\begin{quotation}

			We modelled the relationship between niche overlap and phylogenetic 
			distance using a logistic regression. We used the numbers of shared 
			($M_{ij}$) and non-shared ($U_{ij}$) partners as dependent variables and 
			centred, scaled phylogenetic distance as the independent variable. This 
			approach is conceptually similar to modelling successes and failures in a 
			binomial-distributed process. Accordingly, we assumed a binomially-distributed error structure and used a logit link function to model the dissimilarity in interaction partners 
			$J_{ij}$ of plants $i$ and $j$.

		\end{quotation}


		As stated in the section on phylogenetic data, we excluded comparisons involving plants for which we could not assign a binomial names (and therefore could not obtain phylogenetic distances) and comparisons between non-angiosperms due to our coarse estimates of their ages. This information appeared in lines 107-110 and 118-120 of the previous version and lines 112-115 and 122-124 of the current version.


	13. Request to include plant species as a fixed effect

		\begin{refquote}
			176-178. Why you do not consider using plant species as another fixed factor to break the lack of independence? This is because generalist species may have more weight in your analyses than other species. It would be interesting to see how your results differ from an analysis considering such approach.
		\end{refquote}

		%DIF > DBS: I think your interpretation is incorrect here. I believe that they are referring to the fact that species with many interactions have a greater chance of "strong" overlap than species with few interactions. Either way though, I don't think this can be operationalized because each pairwise overlap value is tied to two species and not one (and that isn't tractable, in a way that I can think of...).
\DIFaddbegin 

		\DIFaddend \textbf{R:} It sounds as though the Referee would like us to include plant species as a predictor in our regressions. Due to the fact that species generally appear in few networks (many in only a single network), this would result in near singularity of our models. We would therefore be completely unable to observe any patterns as species identity would explain nearly all variation in niche overlap. If the Referee is referring to a potential effect of network size, we did in fact test this (see equation 4 and description). If the Referee did not mean either of these possibilities, we apologise but are unable to understand this comment.


	14. Request to include connectance

		\begin{refquote}
			198-209. Why did you decide to control just for network size and do not consider network connectance? You provide methodological information to explain your decision but there is no biological argument explaining why network size may influence the relationship between phylogenetic relatedness and niche overlap.
		\end{refquote}

		\textbf{R:} Our main reason for including network size as a potential effect was methodological rather than biological - slopes for smaller networks are based on smaller sample sizes and therefore may be less robust. \DIFdelbegin \DIFdel{We do not see either }\DIFdelend \DIFaddbegin \DIFadd{In contrast, we were unable to determine among ourselves what }\DIFaddend the methodological or biological arguments \DIFaddbegin \DIFadd{would be }\DIFaddend for including connectance as a predictor
		\DIFdelbegin \DIFdel{. }\DIFdelend We also note that the manuscript is already quite full, so including these new analyses would require us to remove \DIFdelbegin \DIFdel{something }\DIFdelend \DIFaddbegin \DIFadd{which we and other Referees have regarded as }\DIFaddend more important. Without a justification from the Referee as to why connectance should affect pairwise niche overlap, we have opted not to add this new thread to the manuscript.
		%DIF >  DBS: This probably isn't true. If connectance were 0 there would be no overlap and if it were 1 overlap would be perfect. They probably think it sets a useful baseline. (?)


	15. Confusion about motivation for community composition analyses 

		\begin{refquote}
			Line 210 Linking network level trends and community composition

			(related with this part of the methods but referring to the Introduction section) Which are the hypothesis for this analysis? How do you think that family composition (and no species composition) could affect the relationship between phylogenetic  distances and niche overlap?
			In the analysis it is not clear for us which are the dependent and independent variables. Is the independent variable a matrix with coefficients of similarity between networks?  Do you want to explore if more similar networks have a more similar relationship (slope) between phylogenetic distance and niche overlap? Or you want to investigate if the slopes are more o less strong depending on the local composition of communities (for each network separately) ? If this is the case you could compare slopes with a local diversity index (e.g. Shannon) instead of a beta diversity index.
		\end{refquote}


		\textbf{R:} The Referee was correct with their first idea. We want to explore whether more similar networks have more similar slopes of partner overlap against phylogenetic distance. Once again, we are pleased that the Referee agrees with our analysis. We did this because different plant families likely experienced different levels of convergence, coevolution, have different levels of intimacy with their partners, etc. We now explicitly state this in the introduction. Note that we cannot use Shannon diversity within networks as we do not know the abundances of the various plant species. All we can do is test the effect of \DIFdelbegin \DIFdel{numbers of species }\DIFdelend \DIFaddbegin \DIFadd{richness }\DIFaddend as described above.


		Lines 78-86:

		\begin{quotation}

			Here we investigate how overlap in interaction partners between 
			pairs of plants (henceforth ``niche overlap'') varies over 
			phylogenetic distance. 
			Whereas previous 
			studies have focused on the presence or absence of phylogenetic
			signal across entire networks, we take a pairwise perspective in
			order to obtain a more detailed picture of how plant phylogeny
			relates to network structure. As different plant families (which represent tractable clades for analysis) may have experienced different degrees of coevolution, convergence, etc., we also complement analyses with entire networks with comparisons among plants in the same family within a network. 
			This novel perspective allows us to investigate the relationship between phylogenetic distance and partner overlap at different scales. 

  		\end{quotation}


	16. Critique of manuscript structure 

		\begin{refquote}
			Results\\
			General: Please, do not include discussion in your results, it may confound readers.
		\end{refquote}

		\textbf{R:} We are unsure whether the Referee is giving general advice here or if there is a specific element of the results the Referee feels would fit better in the discussion. We \DIFdelbegin \DIFdel{thank the Referee for the advice but can take little practical action without more details as to the nature of their objection.
		}\DIFdelend \DIFaddbegin \DIFadd{have revised the results as best we can to avoid text which could be misconstrued as discussion.
		%DIF > We thank the Referee for the advice but can take little practical action without more details as to the nature of their objection.
}\DIFaddend 


	17. \DIFdelbegin \DIFdel{Incorrect claim that }\DIFdelend \DIFaddbegin \DIFadd{Claim  about accuracy of the }\DIFaddend beginning of results
\DIFdelbegin \DIFdel{is inaccurate
}\DIFdelend 

		\begin{refquote}
			Line 262. Your first sentence of results do not correspond with what follows. We will suggest you use most networks, or most herbivory networks and half pollination networks, or using the percentages you present in the discussion section.
		\end{refquote}

		\textbf{R:} \DIFdelbegin \DIFdel{This comment's somewhat eccentric grammar makes it rather difficult to parse}\DIFdelend \DIFaddbegin \DIFadd{We were unclear regarding what and why the Referee made this comment}\DIFaddend . We are guessing that the Referee is confused by the overall trend (more related plants have more similar interaction partners) and the variability of the trends within specific networks. To help understand this, note that the structure of the first few lines of the discussion \DIFdelbegin \DIFdel{parallel }\DIFdelend \DIFaddbegin \DIFadd{parallels }\DIFaddend the first few lines of the results. First we introduce the overall result, and then we expand upon the details. We have lightly edited lines 370-373 to make this clearer:

		\begin{quotation}

			We found general support for the hypothesis that more
			closely-related pairs of plants have a higher degree
			of niche overlap. Taking all networks together, 
			the probability of two plants sharing the same animal 
			interaction partners decreased with increasing 
			phylogenetic distance. Considering networks separately ...

		\end{quotation}


	18. Definition of community composition unclear


		\begin{refquote}
			Line 310. How was community composition measured? Is difficult to evaluate the quality of this result because you haven't described this methodology.
		\end{refquote}


		\textbf{R:} Community composition is the set of plant species that make up a community, as described by the researchers constructing each network. For tractability, we have summarized community composition as the number of species in each plant family. We have added text to the methods to state this \DIFaddbegin \DIFadd{more }\DIFaddend explicitly.


		Lines 224-233:

		\begin{quotation}
			Next, we examined the connection between our network-level observations
			and the number of species in each plant family present in each community.
			Specifically, we tested the hypothesis that
			varying relationships between phylogenetic distance and
			niche overlap are due to the different distributions 
			of families across networks. We defined the relationship between
			phylogenetic distance and niche overlap as the change in 
			log odds of two plants in a given network sharing an interaction 
			partner per million years of divergence (i.e., the slope $\beta_{distance}$ from the 
			regression of niche overlap against phylogenetic distance within
			a single network). We then related differences in this relationship
			to differences in the composition of the plant community 
			using a non-parametric permutational multi-variate 
			analysis of variance (PERMANOVA;~\citealp{Anderson2001}).
	    \end{quotation}

	    %DIF > DBS: To be fair, I think this might still lack detail. When we say "differences in the composition" in the last sentence, should be say "beta-diversity metric X comparing the family-wise composition of the two networks?"
\DIFaddbegin 


	\DIFaddend 19. Confusion about permutation methodology 

		\begin{refquote}
			Minor comments on Results
			297-298 You permuted interactions or phylogenetic distances?
		\end{refquote}


		\textbf{R:} As stated in the methods, we permuted interactions. Note that, in statistical terms, the two approaches are equivalent, but we appreciate that this may not be clear to readers unfamiliar with permutation approaches. We have rephrased the results to match the methods.


	20. Typo in Fig. 1 caption 

		\begin{refquote}
			Fig 1. "...the probability of a pair of plants sharing an interaction partner increased with increasing phylogenetic distance..." please, write decreased instead of increased.
		\end{refquote}


		\textbf{R:} We thank the Referee for pointing out this truly egregious typo, and have corrected it.


	21. Difference between minimum and maximum slopes not visible 

		\begin{refquote}
			Fig 2. It is no clear for us. Minimum and maximum slopes obtained from 999 permutations are the same value?
		\end{refquote}


		\textbf{R:} As the caption states, the minimum and maximum slopes were always very close to 0. We therefore had to choose between a y-axis scale which would show the observed slopes or one which would show the differences in slopes for the permutations. As the observed slopes are far more important than knowing the exact values of the permutation slopes, we chose to prioritise the former.


	22. Confusion about line 328

		\begin{refquote}
			Line  328: "...increasing phylogenetic distance in 12...", what is 12? the number of families?
		\end{refquote}

		\textbf{R:} Yes, 12 is the number of families. We have revised the line to make this more explicit. Note that we have also corrected this line to match Table 1, which includes 14 significant families.


		Lines 355-356: 


		\begin{quotation}

		In pollination networks, overlap decreased significantly with increasing phylogenetic distance in 14 of the 48 well-represented families (Table 1; Fig. 3).

		\end{quotation}


	23. Complaints about table numbering and figure referencing

		\begin{refquote}
			Table number seem to be erroneous as you first mention table 2 and then table 1.

			\smallskip

			Figure 3 is only referenced in the text when talking about herbivory networks but not for pollination networks.
		\end{refquote}

		\textbf{R:} We have reordered the tables and added a \DIFdelbegin \DIFdel{redundant }\DIFdelend \DIFaddbegin \DIFadd{second }\DIFaddend reference to Fig. 3.


	24. Suggestion to include ideas about evolution and coevolution 

		\begin{refquote}
			Discussion
			General
			We suggest you include ideas about evolution and coevolution in networks in your discussion. It is important to recognize that network structure is in part a consequence of the evolutionary process and that we cannot think in the pairwise-coevolutionary framework as the only one modulating trait evolution in species rich assemblages.
			You state that the fact that the proportion of specialist species may explain differences between plant-pollination and plant-herbivore network patterns and you discuss the mathematical consequences of such difference but you should discuss the ecological and evolutionary causes and consequences of having higher specialization in pollination networks. This is also related with your methodological decision of excluding more specialized herbivores, isn't it?
		\end{refquote}


		\textbf{R:} \DIFdelbegin \DIFdel{We do include ideas about evolution and coevolution in our discussion. We }\DIFdelend %DIF > We do include ideas about evolution and coevolution in our discussion.
		\DIFaddbegin \DIFadd{We }\DIFaddend completely agree that, together with ecological processes and environmental filtering, evolution and coevolution are very important in structuring networks at every level, from pairs of species to entire communities. However, since we do not have trait or genetic data for our networks, we cannot comment with any certainty on which of our results are due to trait plasticity or environmental influence and which are due to evolution. We therefore mention possible evolutionary and ecological explanations together.


		Some examples of evolutionary ideas in the discussion:

		%DIF >  DBS: Aren't you just beating a dead horse below with all the examples? Couldn't a list of the line numbers be sufficient? If I were this reviewer, this would piss me off. Though you might not like them, I seriously doubt that is the image or result you wish to project...
\DIFaddbegin 

		\DIFaddend Lines 413-418:

		\begin{quotation}

			This relationship between overlap and 
			phylogenetic distance is consistent with the idea that traits affecting 
			interactions are heritable and change gradually
			such that closely related plants resemble their common ancestor--- and
			each other ---more than they do distantly related 
			plants~\citep{Schemske1999,Gilbert2015,Ponisio2017}. The degree of 
			heritability of key traits may, however, differ between families.

		\end{quotation}


		Lines 420-427:

		\begin{quotation}

			This could indicate different 
			rates of phenotypic drift or evolution in different families (or their interaction partners). 
			In other families, there was no significant relationship between phylogenetic
			distance and niche overlap. In these cases, key traits affecting 
			plant-insect interactions may be highly labile or plastic (environmentally determined). These possibilities are supported by several studies showing a stronger relationship between niche overlap and trait similarity than niche overlap and phylogenetic similarity~\citep{Junker2015,Ibanez2016,Endara2017}. 

		\end{quotation}


		Lines 432-437:

		\begin{quotation}

			First, part of the family may have recently 
			undergone a period of rapid diversification with closely-related species 
			developing novel phenotypes and attracting different  
			interaction partners~\citep{Linder2008,Breitkopf2015}. Likewise, the
			animals may have undergone an adaptive radiation to 
			specialise on their most profitable partner~\citep{Janz2006}. 
			Alternatively, plants in these families could have undergone convergent evolution or ancestral traits could be strongly preserved.

		\end{quotation}


		Lines 444-446:

		\begin{quotation}

			Selection to avoid 
			competition and restrict numbers of interaction partners may lead to
			more intimate or specialised interactions~\citep{Ponisio2017}.

		\end{quotation}


		Lines 458-466:

		\begin{quotation}

			These plants might 
			be highly specialised on different interaction partners and therefore
			have low overlap at all levels of relatedness. In other plant families
			with more moderate levels of specialisation, it is possible 
			that pollination and/or herbivory do not exert large
			selection pressures on the plants. If traits affecting pollination
			or herbivory are not heritable in these groups [\citealp{Kursar2009}] 
			or their phenotypes are constrained by other factors (e.g., 
			environmental conditions, trade-offs with other traits, ontogenic
			change [\citealp{Karinho2014}]), then we should not expect a relationship 
			between phylogenetic distance and overlap of interaction partners.

		\end{quotation}


		Lines 476-482:

		\begin{quotation}

			This could be because of conflicting selection from pollinators and herbivores,
			with one type of selection placing greater constraints on plant traits than the other.
			Multiple types of interactions (e.g., pollination, herbivory, nectar robbing) 
			and even environmental factors can influence traits such as 
			flower colour, nectar abundance, and flowering phenology~\citep{Strauss2006}. 
			These influences can act in the same or different directions~\citep{Strauss2006}.

		\end{quotation}


	25. Objection to the word "broad"

		\begin{refquote}
			338-339. You state We found broad support for the hypothesis that more closely-related pairs of plants have a higher degree of niche overlap. This is a very strong sentence that is not supported by your results. You found evidence in 55\% of plant-pollination networks (almost half of your dataset). In fact, it is interesting to note that even the slope is more pronounced in mutualistic networks you found higher support in herbivory networks, as a higher percentage of networks showed a tendency.
		\end{refquote}

		%DIF >  DBS: Again, do you really want to pick a fight that is a matter of splitting hairs? Why die in a ditch over our use of the term broad?
\DIFaddbegin 

		\DIFaddend \textbf{R:} \DIFdelbegin \DIFdel{Note that 56\% is greater than half, and that when }\DIFdelend \DIFaddbegin \DIFadd{When }\DIFaddend all networks were considered together\DIFdelbegin \DIFdel{we also }\DIFdelend \DIFaddbegin \DIFadd{, we }\DIFaddend observed the same trends. \DIFdelbegin \DIFdel{We }\DIFdelend \DIFaddbegin \DIFadd{Moreover, 56\% is both (i) greater than half and (ii) much greater than one would expect if the result occurred purely by chance (5\%). Putting this all together, we }\DIFaddend contend that these two facts do represent broad support for our hypothesis\DIFdelbegin \DIFdel{and note that the Hembry et al reference suggested by the Referee uses the word in a very similar way.
		}\DIFdelend \DIFaddbegin \DIFadd{.
		%DIF > and note that the Hembry et al reference suggested by the Referee uses the word in a very similar way.
		}\DIFaddend Broad support is not, of course, universal support. \DIFdelbegin \DIFdel{That }\DIFdelend \DIFaddbegin \DIFadd{This }\DIFaddend is why we provide the numbers of networks\DIFdelbegin \DIFdel{. Readers are thereby armed with the tools }\DIFdelend \DIFaddbegin \DIFadd{, thereby allowing readers }\DIFaddend to agree or disagree with our choice of words. In case it is the synonymy between broad and \DIFdelbegin \DIFdel{general }\DIFdelend \DIFaddbegin \DIFadd{unambiguous }\DIFaddend that has perplexed the Referee, we have changed this line to state ''We found \emph{general} support...''. We hope that this is clearer.
		%DIF >  DBS: If you want to go down this line of argument, I would just leave it as broad....


		We are glad that the Referee finds the greater proportion of significant herbivory networks interesting. With the unbalanced sample sizes of pollination and herbivory networks, however, we are somewhat reluctant to make much of the difference between 56\% and 64\%. We consider that the important point here is that networks are more likely than not to show decreasing niche overlap with increasing phylogenetic distance. 


	26. \DIFdelbegin \DIFdel{Confusing complaint }\DIFdelend \DIFaddbegin \DIFadd{Complaint }\DIFaddend about subsetting herbivory networks or perhaps objection to the claim that oligotrophy is common 

		\begin{refquote}
			351-352 The herbivory networks did not contain as many obligate specialists, but we note that many herbivorous insects are oligotrophs which consume only a few closely-related hosts (Novotny \& Basset, 2005; Yguel et al., 2011). This sentence is confuse because you have a subsample of herbivory networks and then this seems to be a by-product of your study design associated with your decision of removing networks that may show strong phylogenetic signal.
		\end{refquote}


		\textbf{R:} We must again point out that subsetting the herbivory networks as we have done was requested during an earlier round of review. We are therefore reluctant to reconsider this approach without a very good reason. 


		Beyond that, it is very difficult to work out exactly what the Referee objects to in this sentence. If the Referee is objecting to our references, note that~\citet{Novotny2005} addresses leaf-chewers as intermediate in their level of specialisation on particular families and emphasises that herbivores in general (regardless of guild) tend to consume a few closely-related hosts. They point out the relative rarity of strict specialists compared to specialists on species within a particular genus or family.~\citet{Brandle2006} also found insect herbivores were specialised on a particular genus but consumed multiple species within the genus. As~\citet{Brandle2006} focus on phytophagous insects specifically, we feel that this reference addresses our point more directly than~\citet{Yguel2011}. We have therefore replaced this reference. We hope that this improved selection of references will help the Referee's confusion.


		If the Referee objects to our finding that the herbivores in our networks tended to consume more than one plant species, there is little we can do about it. That is simply what our data say. 


		If the Referee is contending that, because of the multiple herbivore types in~\citet{Novotny2005}, we cannot claim that the herbivores in our dataset are oligotrophs, we can respond with further details. The mean number of plants consumed per herbivore ranged between 1.64 and 14.9 across networks, with a mean of 6.36, and the median ranged between 1 and 11.5 (mean 4.68). These measures indicate that many herbivores in our networks do, in fact, consume only a few of the plants available to them (with some highly generalist species increasing the means). Our overall point, that oligotrophy may be common in herbivores, therefore \DIFaddbegin \DIFadd{likely }\DIFaddend stands.


	27. Claims that congeneric species are not closely related 

		\begin{refquote}
			357-359 This suggests that conservation of interaction partners among closely related plants (e.g., congeners or members of the same subfamilies) is more important than phylogenetic signal from deeper within the phylogenetic tree. Congeneric species can be not closely related, please revise sentences like this one across the manuscript because you are confounding phylogeny with taxonomy.
		\end{refquote}

		%DIF >  DBS: Tone it down! There is *nothing* to be gained from the response below. It would be easier and more prudent to just say that taxonomy is an imperfect proxy of phylogeney and yes, sometimes taxonomic congeners might actually be more distant phylogenetically though on the whole the match is pretty damn good. 
\DIFaddbegin 

		\DIFaddend \textbf{R:} Does the Referee really mean to suggest that plants in the same genus are generally less closely-related than plants in different families? Without some sources to substantiate this, we find this claim very counter-intuitive. This is especially surprising given the Referee's insistence that we should expect similar interaction partners within a family. We agree that taxonomy is only a proxy for phylogeny, but we are not aware of any evidence that it is a poor proxy. We also note A) that similar language is quite common in the literature and B) that across seven other reviews, no other Referee has suggested that species in the same taxonomic groups (e.g., genera) are less closely-related than species in different groups. Without any evidence to support this claim, we have opted not to change our text.


	28. \DIFdelbegin \DIFdel{Another demand }\DIFdelend \DIFaddbegin \DIFadd{Request }\DIFaddend to discuss intimacy [[done. Look for more places to scattershot Hembry]]

		\begin{refquote}
			361-365. How interaction intimacy may explain your results? Please, discuss your results in light of the interplay between interaction type (pollination, herbivory) and intimacy (see Hembry et al. 2018).
		\end{refquote}


		\textbf{R:} We do not believe that interaction intimacy has a strong effect on our results, as both leaf-chewing herbivory and pollination are low-intimacy interactions~\citep{Astegiano2017}. It is possible that there is variation in intimacy within these broad categories, but we \DIFaddbegin \DIFadd{unfortunately }\DIFaddend do not have data that would allow us to test such a hypothesis. We also note that~\citet{Hembry2018} study a very small and high-intimacy system (a single genus of brood pollinating moths and their single genus of host plants). Their results will therefore not necessarily generalize to the larger, more diverse networks we consider \DIFaddbegin \DIFadd{here}\DIFaddend . Indeed, had they considered a broader sample of plants and seed predators or herbivores from the same islands, they easily could have found less reciprocal specialisation. As a detailed study of this unusual mutualism was the aim in~\citet{Hembry2018} this is not a problem for their work, but we are hesitant to infer too strong a relationship between their findings and ours.


		This reference suggestion is also somewhat confusing as~\citet{Hembry2018} did not find evidence for host-associated differentiation in their moths. This would seem to undermine the idea that evolution/coevolution can explain network structure. Moreover, they point out that the support for the intimacy hypothesis  in their system was broad (in the same sense we used the word) but equivocal. This exacerbates the difficulty \DIFdelbegin \DIFdel{of }\DIFdelend \DIFaddbegin \DIFadd{we found in }\DIFaddend extrapolating their work to explain our results.


		Given all of the above, it is hard to see how the interaction intimacy hypothesis relates to these lines (which are about whether the number of species in different plant families is related to the strength of phylogenetic signal). Instead, we \DIFdelbegin \DIFdel{feel }\DIFdelend \DIFaddbegin \DIFadd{felt }\DIFaddend that a discussion of the possible effects of intimacy \DIFdelbegin \DIFdel{is }\DIFdelend \DIFaddbegin \DIFadd{was }\DIFaddend more relevant later on, where we discuss possible reasons why some families may show different trends. This section (lines 443-454) now reads:


		\begin{quotation}

			Indeed, animal pollination and seed dispersal have been shown to act
			as filters for several plant clades~\citep{Mayfield2009}. Selection to avoid 
			competition and restrict numbers of interaction partners may lead to
			more intimate or specialised interactions~\citep{Ponisio2017}. 
			Reciprocal specialisation is particularly expected in highly intimate interactions~\citep{Hembry2018} rather than the relatively low-intimacy pollination or leaf-chewing herbivory we consider here~\citep{Astegiano2017}.
			Despite this, it is still possible that some plant families may tend to have more intimate interactions than others. This could result, for example, if a plant is pollinated by an insect which is also a herbivore in its larval stage~\citep{Hembry2018}. 
			Whether or not intimacy plays a role in structuring these interactions, past selection to avoid competition is 
			consistent with the relatively high proportion of extreme specialists we
			observed in the pollination networks.

		\end{quotation}


\clearpage

{\Large \bf Reply to Referee 3} 

	\begin{refquote}
		The study is well designed and the manuscript well written, and the authors clearly demonstrate their comprehensive grasp of the subject and the applied statistical analyses.
		My only more general comment is whether authors have considered potential applications of such phylogenetic approaches for ecological restoration? If closely-related species provide similar functions in ecological networks, they could be used interchangeably to restore ecological interactions/functions. This is an exciting potential application of phylogenetic data in biodiversity conservation and restoration, and could be touched upon/discussed in this manuscript.
		Otherwise, I have only minor comments.
	\end{refquote}

	\textbf{R:} We thank the Referee for their compliments to our approach and to our writing, as well as their more detailed comments (addressed one-by one below; replies preceded by \textbf{R:}). We had not considered this potential application for ecological restoration but are very pleased to have it pointed out. It is often difficult to see how theoretical work may relate to conservation practice, so we are glad to take the opportunity to make a link here. We have reframed our closing paragraph to introduce the possibilities for using closely-related species in restoration efforts, and believe that it emphasises the importance of continuing research along this line. We hope that the Referee agrees that introducing this idea makes for a much stronger conclusion.


	Lines 495-511:

	\begin{quotation}

		Altogether, our study has revealed general trends for conservation of interaction
		partners between closely-related species, with some networks and plant 
		families showing different trends. This overall similarity between closely-related
		species has a potential application in ecological restoration. Close relatives could
		be used interchangeably to restore missing interactions and fill ecosystem functions. 
		This may be advantageous when a target plant is more difficult to establish than its
		relatives, or if the restoration site is not large enough to support viable populations 
		of many species. We should urge caution, however, since plants which support the
		same pollinators may also support similar sets of herbivores. To avoid unwanted 
		indirect effects, all interactions involving the target species should be considered.
		Although here we considered only the presence or absence of interactions,
		(i.e., qualitative networks)
		recent work also suggests that the phylogenetic composition of a plant
		community can also affect the strength of 
		interactions, and that the spatial arrangement of plants within a 
		community may be particularly important~\citep{Yguel2011,Castagneyrol2014}.
		These further nuances in the relationship between phylogenetic distance and 
		niche overlap could also strongly affect the ability of closely-related species to
		fill the same functions in restoration efforts. This is clearly a topic with many
		unresolved questions, deserving of further study.

	\end{quotation}


	1. Title is confusing

		\begin{refquote}
			Title is confusing – difficult to pick out key message. And given that your analyses broadly give support for effects of phylogenetic distance on niche overlap, why not focus on this message, rather than variability at among individual networks/plant families
		\end{refquote}


		\textbf{R:} We have changed the title to state our main result while also pointing out some of the important variability between networks. We do feel that this variability is a major feature of our results, as well as previous studies of phylogenetic signal across systems, and it is too easy to forget if not emphasized. We hope that the Referee will agree that our new title, "Related plants tend to share pollinators and herbivores, but strength of phylogenetic signal varies among plant families", strikes a balance between these two considerations.


		% "At a global scale, conservation of pollinators and herbivores between related plants varies widely across communities and between plant families.”

		% - suggests that we focus on broad support. Maybe "Broad conservation of pollinators and herbivores in a global dataset"?
		% "Despite variation among families, pollinators and herbivores are broadly conserved among related plants"?
		% "Related plants tend to share pollinators and herbivores, but strength of phylogenetic signal varies among plant families"?

	2. Request to state that networks are qualitative rather than quantitative 

		\textbf{R:} The Referee is correct that the difference between qualitative and quantitative networks is crucial for the correct ecological interpretation of said networks. We thank them for pointing out that the distinction wasn't clear in the previous version. We now specify that our networks are qualitative at the beginning of the methods and results and the end of the discussion. We hope that these few reminders will be enough to remind readers about the structure of our networks.

		Lines 105-106 (methods):

		\begin{quotation}

			All networks were qualitative and did not include interaction strengths.

		\end{quotation}


		Lines 294-296 (results):

		\begin{quotation}

			Note that, as our networks 
		    are qualitative, these results refer only to the number of shared interaction
	    	partners rather than to the quantitative strength of competition.

    	\end{quotation}


    	Lines 504-505 (discussion):

    	\begin{quotation}

		  Although here we considered only the presence or absence of interactions,
		  (i.e., qualitative networks) ...

    	\end{quotation}


	3. L298 – Please be careful with use of “lesser/greater” “more negative or positive than expected from permuted networks” 


		\textbf{R:} We appreciate that this description could be unclear, and have revised this line as suggested. It now reads:

		Lines 326-328:

		\begin{quotation}

			the observed 
			slope of the relationship between phylogenetic distance and interaction 
			partner overlap was always more extreme (i.e., always more negative or 
			always more positive) than that obtained in the permuted networks (Fig. 2).

	    \end{quotation}


	4. Typo in line 322 

		\begin{refquote}
			L322 – how was this result obtained if insect phylogenies were not calculated? Do you mean “more closely-related plants in pollinator networks”?
		\end{refquote}


		\textbf{R:} The Referee is correct and we apologize for this extremely confusing typo. We have corrected this line as suggested.


\clearpage


{\Large \bf Reply to Referee 4} 

	\begin{refquote}
		Understanding what drives the structure of networks of interactions is key to predict the structure of plant-insect communities (e.g., for conservation or restoration purposes, or to support benefit organisms in agriculture).
		The present study focuses on the contribution of the evolutionary history to the interaction structure, and makes a significant contribution to the understanding of the variability of the phylogenetic signal in plant-insect networks. Here, the strength of the phylogenetic signal is actually quantified while it usually relies on correlation scores, and the effect of plant composition and the identity of plant families are examined to assess whether they could explain the variability of trends described in the literature.
		The dense writing turns out to be a double-edge sword: a lot of information on drivers of phylogenetic signals is compiled in here, but the text is sometime hard to follow. Hence, my following comments mostly pertains to the form, and I trust the authors should be able to address them fairly easily.
	\end{refquote}


	\textbf{R:} We thank the Referee for their compliments to our analysis and for taking the time to review our manuscript so thoroughly. We acknowledge that our writing can be dense at times and appreciate the Referee pointing out places where we may clarify our prose. We especially appreciate the detailed suggestions for how to improve our second-to-last paragraph, and have followed them to the best of our ability. Below, we respond to each comment in detail (preceded by \textbf{R:}). We are confident that revising the manuscript in response to the Referee's comments has significantly improved it.


	1. Confusion about description of tuple-form dependent variable

		\begin{refquote}
			* Modelling the relationship between niche overlap and phylogenetic distance\\
			I did not managed to fully grasp the statistical analysis right away (I reckon I'm not a statistical master, but this may be the case of other readers), and I think it was mostly due to the way the methods are described. The writing is nice, but some tiny details are missing while they could help the reader to understand more quickly how the authors distinguish the use of the {M\_ij, U\_ij}-tuple from that of J\_ij.
			Based on l. 143-167, I understand that the number of shared interactions M\_ij is modelled with a binomial distribution, with the number of trials being M\_ij+U\_ij, and the probability of success being w\_ij (M\_ij ~ B(M\_ij + U\_ij, w\_ij)). The {M\_ij, U\_ij}-tuple would appear in R as follows:
			model $\leftarrow$ glm(cbind(M\_ij, M\_ij+U\_ij) \~ d\_ij + ..., family = “binomial”)
			But, can we actually say that there are two dependent variables? To me, it is the number of shared interactions that is modelled as a binomial process, not both the number of shared and unshared interactions.

			\smallskip

			p. 5, l. 69-71 « In either case, dissimilarity (…). » This sentence helped me to understand why it is important to look at unshared interactions. To be better showcased ?

		\end{refquote}


		\textbf{R:} The number of shared and unshared interactions are indeed both dependent variables included in the analysis. The number of shared variables is the key variable, but the number of unshared interactions is also important as it indicates how extraordinary a particular number of shared interactions is. The binomial process incorporates both quantities; converting the numbers of shared and unshared interactions to a proportion and weighting the observations by the total number of interactions. We thank the Referee for pointing out text in the introduction which clarified the purpose of including unshared interactions, and have used it as the basis for improving the methods. We have expanded the beginning of this section to explain the logic of our modelling approach in more detail.


		lines 130-141:


		\begin{quotation}

			We calculated niche overlap for each pair of plants $i$ and $j$ based on the number of shared and unshared interaction partners ($M_{ij}$, $U_{ij}$, respectively). 
			The number of unshared interaction 
			partners gives valuable information about cases where, for example, 
			closely-related plants may have experienced disruptive selection, leading to weaker phylogenetic signal. 
			The sum $M_{ij} + U_{ij}$ indicates the amount of information  provided by each pair of plants: a pair of generalists which share most of their interaction partners gives a stronger indication of phylogenetic signal than a pair of extreme specialists with one common interaction partner.


			Together, $M_{ij}$ and $U_{ij}$ give a Jaccard index ($J_{ij}$) describing 
			the proportion of shared interactions. $J_{ij}$ is defined: 
			\begin{equation}
			J_{ij} = \frac{M_{ij}}{U_{ij}+M_{ij}} ,
			\end{equation}
			where $M_{ij}$ is the set of \emph{mutual} (shared) interaction partners and $U_{ij}$ the set of unshared interaction partners for plants $i$ and $j$.

		\end{quotation} 


	2. Request to state that we use the Jaccard index more explicitly 


		\begin{refquote}
			I suppose that the analysis described in paragraph starting from l. 168 corresponds to actually modelling the Jaccard index (J\_ij \~ ...). If this is right, I think this should be specified in the second paragraph (no direct mention of the Jaccard index is currently made, except the proportion l. 170 between brackets although it is not exactly written as in eq. (1)), while in the first paragraph, an equation should specify that M\_ij ~ B(M\_ij + U\_ij, w\_ij)).
		\end{refquote}


		\textbf{R:} Yes, this analysis is also using a Jaccard index, but we see how the previous version did not make this clear. We now state that both the tuple and proportional glm's use Jaccard dissimilarity. We also re-iterate the logic behind the tuple-based formulation in this paragraph.


			Lines 173-181:


			\begin{quotation}

				To demonstrate the power of defining $J_{ij}$ as a tuple of $M_{ij}$ and $U_{ij}$ rather than a single value, we repeated the above analyses using a Jaccard index based only on the proportion of interaction partners that are shared (i.e., $J_{ij}$ = $M_{ij}$/[$M_{ij}+U_{ij}$]). Note that while the proportion of shared interaction partners is the same in both cases, the tuple formulation gives more weight to plants with many interaction partners as these provide more information. When comparing the two approaches 
				we observed similar trends but, notably, the tuple definition of $J_{ij}$ had greater power to detect weak relationships (\emph{Supporting information 2}). We therefore show only the results when defining $J_{ij}$ as a tuple in the main text.

			\end{quotation}


	3. Request to make non-independence of species within each community explicit. 

		\begin{refquote}
			Another point pertains to the definition of “trials” in the binomial-distributed process. If I get it right, these correspond to the union of species i and j interactions (of size P\_i + P\_j -2M\_ij = M\_ij + U\_ij). However, these are not independent for reasons highlighted in the introduction (e.g., competition, facilitation between plants and/or insects). This may question the robustness of the analysis, but testing the effect of the plant community composition (negative) allows to verify this. This should be specified, I think.
		\end{refquote}


		\textbf{R:} We actually address the non-independence of trials (pairs of plants) by calculating $p$-values based on permutations of the observed networks (as opposed to assuming independent, binomially-distributed trials, which is certainly not the case). To emphasize our explanation of this, we have reordered the text and placed our discussion of the permutation methods under the sub-heading 'accounting for non-independence'. We hope that this will calm reader's worries about the validity of our conclusions.


	4. Permuting the permuted networks is confusing

		\begin{refquote}
			* Evaluating type I and type II errors: Permutations of permutations\\
			Paragraph p. 10, l. 187-197 is a bit confusing because of phrasings such as “permutations/permuting of the permuted networks”. I don't have a better way to name this second round of permutations, but the overall is difficult to follow and could be shortened while still conveying the same message.
			Actually, I find the comment in Supplementary information 3 clearer, although no reference to Supplementary information 3 is made.
			Besides, as slopes of regressions in permuted networks are tightly grouped around zero (Fig. 2, thin lines indicating 
			extrema of slopes distribution overlapping each other), so type II errors do not seem to be a big problem here.
		\end{refquote}


		\textbf{R:} We agree with the Referee about the difficulty of describing this procedure. We have added an extra line to further explain the logic behind the second round of permutations and hope that this will clarify the goal of these analyses. We have also added a reference to Supplementary Information 3 and thank the Referee for pointing out the oversight.


		Lines 209-212:

		\begin{quotation}

			This permutation approach also allows us to estimate type I and type II 
			error for our analysis. Because the permuted networks should not demonstrate any particular relationship between phylogenetic distance and partner overlap, these slopes should be similar to those obtained after permuting these networks a second time.

		\end{quotation}


	5. Suggestion to consider phylogenetic diversity instead of number of plant pairs

		\begin{refquote}
			* Testing the effect of network size on the existence of a phylogenetic signal\\
			Here, it is specifically the number of plant pairs that is studied. I wonder whether the phylogenetic diversity of plants would not be more adapted here. Is it overlooked because similar among networks?
			I think knowing more about this effect would further illuminate why the strength of the phylogenetic signal varies between networks and would be complementary to the analysis testing the effect of plant community composition.
		\end{refquote}


		\textbf{R:} We agree that phylogenetic diversity would be another option for quantifying the 'size' of these networks. However, defining the phylogenetic diversity of each network would depend strongly upon the correctness of of base tree. As we discuss in the methods, our estimates for non-angiosperms are likely to be fairly rough, and some other species are also only approximately dated. Because of this, and because this analysis would more compliment our community-composition analysis than introduce new material, we have opted no to pursue phylogenetic diversity in this manuscript. Here, we were more concerned with numbers of plant pairs since numbers of species are known to affect many areas of network structure and because the number of plant pairs is effectively the sample size for our within-network analyses. 


	6. Request for consideration of multiple testing

		\begin{refquote}
			* Testing within-family existence of a phylogenetic signal\\
			Examining the strength of phylogenetic signal within plant families is an interesting idea. However, this leads to multiple testing which is prone to type I errors and I could not find mention of correction for this multiple testing in the text. There are actually high chances that there is at least one significant test in each table (1 – (1 – alpha)\^N = 0.36 and 0.91 respectively, alpha = 0.05). Yet, the authors can indicate that there are low chances to have that many significant phylogenetic signals (using the probability mass function of the Binomial distribution, see Moran 2003 in Oikos).
		\end{refquote}


		\textbf{R:} The Referee makes a very good point. We had considered these high numbers of significant slopes unlikely to occur by chance, even if we would expect one or two significant results by chance. This argument is somewhat difficult to make convincingly and briefly in the middle of the manuscript, however. Instead, we apply the sequential correlated Bonferroni test in~\citet{Drezner2016}. This development of this test was heavily motivated by the Moran paper you recommend, and we thank the Referee for pointing out this literature. We have added tables of the SCBT critical values to \emph{Supplemental Information 4} in order to provide readers with a bit more detail. We have also added a few lines to the methods to formally introduce the problem of multiple testing, some of the objections to the strict Bonferroni in~\citet{Moran2003}, and our solution to it.


		Lines 278-285:

		\begin{quotation}

			By considering each family separately, we do risk obtaining some significant results purely by chance. The standard technique for addressing this type of multiple hypothesis testing, the Bonferroni correction, tends to be over-zealous and lead to a failure to reject the null hypothesis even when a large number of significant results before the correction supports the alternative hypothesis~\citep{Moran2003}. To account for multiple testing while also allowing the number of families showing significant trends to carry some weight, we use the correlated Bonferroni test introduced in~\citet{Drezner2016} (\emph{Supplemental Information 4}).

		\end{quotation}


		Lines 355-360 now read:

		\begin{quotation}
			 In pollination networks, overlap decreased significantly with increasing phylogenetic distance in 14 of the 48 well-represented families (Table 1; Fig. 3). If we apply the correlated Bonferroni correction to account for multiple testing~\citep{Drezner2016}, all of these slopes remain significant (\emph{Supplemental Information 4}).
		\end{quotation}


		In responding to this comment, we also noted that the number of significant families in the pollination networks was incorrectly quoted in the main text. As shown in Table 1, there are actually 14 significant families in the pollination web. We have corrected this error and thank the Referee again for calling our attention to this section of the results.


	7. Suggestion to mention sampling completeness as a potential explanation for specialists


		\begin{refquote}
			On the effect of specialist pollinators\\

			I think that Reviewer \#3 made an interesting remark on specialist pollinators, and I agree with the authors on the impossibility to assess sampling completeness on the network of this database. However, I think this should be mentioned in the text, at least to invite future works to provide their sampling effort when publishing new empirical data (hence, helping to improve future analyses).
		\end{refquote}


		\textbf{R:} We share the Referee's desire to prompt researchers to share their sampling effort, and have added a short mention of the possibility  that some of our specialists are actually just rare.


		Lines 386-391:


		\begin{quotation}

			Note that some of the apparent specialists in our dataset may actually be rare species involved in more interactions which have not yet been observed~\citep{Bluthgen2006,Poisot2015}. Without information on the sampling completeness of the networks in our dataset, it is difficult to estimate the size of this effect. 
			It is possible, however, that we might observe stronger relationships between phylogenetic
			distance and niche overlap with more complete data on rare species.

		\end{quotation}


	8. Discussion of potential tradeoffs between interactions could be sharpened


		\begin{refquote}
			On plant families involved in different types of interactions\\

			p. 19-20, l. 432-448: This paragraph on systems for which both pollination and herbivory were sampled is a bit fuzzy. Talking about how diverse types of interactions can work together is a nice way to bring back the study to a broader context, though.
			First, I think the sentence of simultaneous selection pressure of pollination and herbivory could be a bit clarified (l. 439-442, “Plants may not be able to respond (…).”). Strauss \& Whitthall’s chapter in the book “Ecology and evolution of flowers” gives a good insight on how these selection pressures can act together or antagonistically in shaping plants phenotypes (and eventually interactions, although not specifically discussed in their chapter).
			Second, the structure of interaction patterns could be discussed with more references than Sauve et al. (2016). Astegiano et al. (2017) make an interesting point on the asymmetry of interaction patterns in such systems (using Pearse and Altermatt (2011)’s data on Lepidotera diet, and citing the Norwood farm data set of Pocock et al. 2012, and the Donana network in Melian et al. 2009).
			Finally, the existence dynamical drivers of the structure of pollination and herbivory is mentioned. Sauve et al. (2014, 2016) indeed suggest that the way pollination and herbivory are distributed on the plant community may matter for community stability, and they studied one case where this structure is indeed not random. Yet, they do not say that these communities should be more stable (compared to which ones since plants undergo both herbivory and pollination in nature?). Whether the interconnection structure arises because it provides greater stability (as discussed by Sauve et al. 2016), or as a by-product of the assembly process (Maynard et al. 2018, Ecol. Lett.) remains to be discussed for networks combining different types of networks.
			Despite the length of this last comment, I don’t mean that the authors should develop proportionally this paragraph, but rather sharpen it.
		\end{refquote}


		\textbf{R:} We thank the Referee for their suggestions and for the small reading list. We have revised this paragraph along the lines above and hope that it is now clearer. We have expanded our discussion of (potentially) conflicting selection from different interaction types and removed our brief reference to dynamical drivers. We believe that sticking more closely to a single theme has helped to focus this paragraph. It now reads:


		\begin{quotation}

			For those few families which were well-represented in \emph{both} pollination
			and herbivory networks, we can also contrast the 
			trends in the two network types. Notably, all families except \emph{Asteraceae} 
			showed different trends in different network types. 
			This could be because of conflicting selection from pollinators and herbivores,
			with one type of selection placing greater constraints on plant traits than the other.
			Multiple types of interactions (e.g., pollination, herbivory, nectar robbing) 
			and even environmental factors can influence traits such as 
			flower colour, nectar abundance, and flowering phenology~\citep{Strauss2006}. 
			These influences can act in the same or different directions~\citep{Strauss2006}.
			Plant phenotypes in turn affect which species participate in both pollination and herbivory~\citep{Strauss1997,Strauss2002,Adler2004,Adler2006,Theis2006}.
			The interplay between these different selective pressures may mean that plants
			cannot evolve to respond optimally to both pollinators and herbivores. Put another
			way, stronger selective pressure from herbivores might cause phenotypic changes
			that disrupt phylogenetic signal in pollinators, or vice versa. This could result from
			asymmetric degree distributions: within a single system, most plants tend to interact
			with many pollinators \emph{or} many herbivores but not both~\citep{Melian2009,Pocock2012,Astegiano2017}.
			These asymmetric interactions may also affect higher-order network structures such as
			modularity or nestedness~\citep{Astegiano2017}. The nature of the effects of multiple interaction types on both phylogenetic signal in interactions and overall network structure is, however, still an open question deserving of much more research.

		\end{quotation}


	Minor comments


	9. Request for additional detail in summary point 2

		\begin{refquote}
			In the abstract, item 2 « shared and unshared interactions »: if it does not breach the word limit, I think a few words specifying why this is important would better showcase the study, and help to understand how it contributes to the study novelty.
		\end{refquote}

		\textbf{R:} We appreciate the Referee's suggestion. Although we did not have much room to expand the summary, we have added a short parenthetical explanation of why considering both shared and unshared partners is helpful. This point now reads:


		\begin{quotation}

			\begin{itemize}
			\item We quantify overlap of interaction partners for all pairs of plants in 59 pollination and 11 herbivory networks based on the numbers of shared and unshared interaction partners (thereby capturing both proportional and absolute overlap). We test 1) for relationships between phylogenetic distance and partner overlap within each network, 2) whether these relationships varied with the composition of the plant community, and 3) whether well-represented plant families showed different relationships. 
		\end{itemize}

		\end{quotation}


	10. Incorrect referencing of Sauve et al., 2016

		\begin{refquote}
			p.3, l. 1-2, citation of Sauve et al. 2016 (Ecology) to say that interactions between plants and animals are critical to plants’ life cycle. I’m not sure this is the best pick as Sauve et al. rather look at how pollination and herbivory interactions are distributed in the plant community and how it may affect community stability. The following references are adequate though.
		\end{refquote}


		\textbf{R:} We have removed this reference as suggested.


	11. Possible typo in page 3, lines 9-10

		\begin{refquote}
			p. 3, l. 9-10 « A plant’s traits are also (...) », I think there is a typo here. Shouldn’t « a » be removed ?
		\end{refquote}


		\textbf{R:} We do not believe that the original formulation was incorrect, but agree that it was inelegant. We have revised this line to read:


		\begin{quotation}
			Plant traits are also likely to determine \emph{which} specific pollinators 
			  and herbivores interact with a particular plant.
	  	\end{quotation}


	  	We hope that this revised version reads better.


	12. Suggestion to remove long list of citations in lines 22-25


		\begin{refquote}
			p. 3, l. 22-25 : There is a lot of citations here. I agree they all point to « mixed results » but this pertains to different aspects of phylogenetic signal. Some highlight differences between mutualistic and antagonistic systems, other rather focus on difference of phylogenetic signal between trophic levels, or even focus on subset of interaction networks. I think, it would be more relevant to the reader to cite these different works to explain how mixed results are (as done in the text following this long citation list).
		\end{refquote}


		\textbf{R:} We had intended these citations to showcase a selection of researchers addressing different aspects of phylogenetic signal, but appreciate how including them all together could be confusing. We have removed the large list so as not to distract from the more specific citations in subsequent lines.


	11. Better showcase sentence on page 5, lines 78


		\begin{refquote}
			p. 5, l. 78 « a pairwise perspective » : This contributes to the manuscript novelty. Maybe this should be further specified?
		\end{refquote}


		\textbf{R:} We thank the Referee for noticing this line, and have added a bit more detail as to how this pairwise perspective allows us to consider both whole-network trends and trends within families.


		Lines 79-86:


		\begin{quotation}

			Whereas previous 
			studies have focused on the presence or absence of phylogenetic
			signal across entire networks, we take a pairwise perspective in
			order to obtain a more detailed picture of how plant phylogeny
			relates to network structure. As different plant families (which represent tractable clades for analysis) may have experienced different degrees of coevolution, convergence, etc., we also complement analyses with entire networks with comparisons among plants in the same family within a network. 
			This novel perspective allows us to investigate the relationship between phylogenetic distance and partner overlap at different scales. 

		\end{quotation}


	12. Suggestion to make a figure showing distribution of webs, or at least to add countries to the list in SI


		\begin{refquote}
			p. 6, l. 89-90 « These networks span a range of biomes (…). » In supporting information, the countries are not indicated in the list. Alternatively, and to emphasize on the title following Reviewer \#2 suggestion in the previous round of review, a figure showing the spatial distributions of the 59 + 11 network on a world map would be welcome here.
		\end{refquote}


		\textbf{R:} We have added the countries or regions (e.g., islands associated with mainland countries, such as Greenland and Denmark) to Table 1 in the SI. We hope that this will provide enough clarification about the spatial scale of our dataset. For readers who are interested, we now refer both to the original sources and from an online database (the Web of Life dataset) which collects the pollination networks we used. These sources can be used to obtain more exact coordinates. 


	13. Suggestions to clarify explanation of regressions 

		\begin{refquote}
			p. 7, l. 125-141 : I think the beginning of this section should be more straightforward and contrast better the information provided by M\_ij and U\_ij (starting with them) with that of the Jaccard index J\_ij. In addition, J\_ij is never mentioned again as such (but is an estimate of w\_ij).
		\end{refquote}


		\textbf{R:} We have reworded this section in an attempt to clarify the relationships between $M_{ij}$, $U_{ij}$, and $J_{ij}$ and better contrast the information given by each value. We have also changed all $\omega_{ij}$ to $J_{ij}$ for simplicity. This paragraph (lines 130-146) now reads:


		\begin{quotation}

			We calculated niche overlap for each pair of plants $i$ and $j$ based on the number of shared and unshared interaction partners ($M_{ij}$, $U_{ij}$, respectively). 
			The number of unshared interaction 
			partners gives valuable information about cases where, for example, 
			closely-related plants may have experienced disruptive selection, leading to weaker phylogenetic signal. 
			The sum $M_{ij} + U_{ij}$ indicates the amount of information  provided by each pair of plants: a pair of generalists which share most of their interaction partners gives a stronger indication of phylogenetic signal than a pair of extreme specialists with one common interaction partner.


			Together, $M_{ij}$ and $U_{ij}$ give a Jaccard index ($J_{ij}$) describing 
			the proportion of shared interactions. $J_{ij}$ is defined: 
			\begin{equation}
			J_{ij} = \frac{M_{ij}}{U_{ij}+M_{ij}} ,
			\end{equation}
			where $M_{ij}$ is the set of \emph{mutual} (shared) interaction partners and $U_{ij}$ the set of unshared interaction partners for plants $i$ and $j$.
			In our statistical analyses (see below), we used the tuple ($M_{ij}$, $U_{ij}$) as the
			dependent variable rather than the single value $J_{ij}$. 
			This allows us to preserve information about the amount of information provided by each pair of plants and weight the observations accordingly.
			Note that species sharing a large \emph{number} of interaction partners may not share a large \emph{proportion} of interaction partners if the number of interaction partners that are not shared is also large.

		    \end{quotation}


	14. Rephrasing of permutation strategy confusing

		\begin{refquote}
			p. 14, l. 297-298 “Comparing the results in the observed networks with those obtained after permuting phylogenetic distances across pairs of plants (...)”: In the methods, interactions are shuffled, not phylogenetic distances. Doing one of the other seems equivalent for this test, but this rephrasing is a bit confusing.
		\end{refquote}


		\textbf{R:} We thank the Referee for pointing out this rephrasing, and have corrected the results to read the same as the methods.


	15. Typo in page 15, lines 322-324


		\begin{refquote}
			p. 15, l. 322-324 “ More closely-related pollinators did, however, tend to share fewer interaction partners (…).”: This is the first time that conservatism of interactions is checked from the pollinators perspective in the manuscript. Is it a typo or did I miss something in the manuscript?
		\end{refquote}


		\textbf{R:} Yes, this is indeed a typo. We did not have information on pollinator phylogenies and did not test anything from the pollinator perspective. We thank the Referee for pointing this out, and have corrected the line. The sentence now reads:


		Lines 350-352:


		\begin{quotation}
			Plants in pollination networks did, however, have a lower intercept probability of sharing interaction partners ($\beta_{pollination}$=-0.776, $p$=0.007), similar to our within-network results above.
		\end{quotation}


	16. Fig. 3, Table 2 conflict for Poaceae


		\begin{refquote}
			Legend of Fig. 3 “(…) Melastomatoaceae and Poaceae in pollination networks are not significantly different from zero.” but Table 2 shows the opposite for Poaceae.
		\end{refquote}


		\textbf{R:} We thank the Referee for pointing out this error. The table is correct, and the change in log odds for \emph{Poaceae} in pollination networks was significant. We have corrected the legend of Fig. 3.


	17. A confusing attempt at subtlety

		\begin{refquote}
			In the figures, $\beta_{distance}$ is alternatively named “slope of regression line” (Fig. 2) and “change in log odds” (Fig. 3).
		\end{refquote}


		\textbf{R:} $\beta_{distance}$ is both the slope of the regression line \emph{and} the change in log odds of a pair of plants sharing a partner as distance increases. We wanted to include both interpretations to help readers relate the regressions to what is happening ecologically. However, we can see how having the different interpretations isolated in different figures could be confusing. We have therefore added parenthetical definitions in both captions to make it clear that Fig. 2 and Fig. 3 are referring to the same quantity.


	% References

	% Altermatt, F. and Pearse, I.S., 2011. Similarity and specialization of the larval versus adult diet of European butterflies and moths. The American Naturalist, 178(3), pp.372-382.

	% Check. % Astegiano, J., Altermatt, F. and Massol, F., 2017. Disentangling the co-structure of multilayer interaction networks: degree distribution and module composition in two-layer bipartite networks. Scientific reports, 7(1), p.15465.

	% Check. % Maynard, D.S., Serván, C.A. and Allesina, S., 2018. Network spandrels reflect ecological assembly. Ecology letters, 21(3), pp.324-334.

	% Check. % Melián, C.J., Bascompte, J., Jordano, P. and Krivan, V., 2009. Diversity in a complex ecological network with two interaction types. Oikos, 118(1), pp.122-130.

	% Check. % Moran, M.D., 2003. Arguments for rejecting the sequential Bonferroni in ecological studies. Oikos, 100(2), pp.403-405.

	% Check. % Strauss, S.Y. and Whittall, J.B., 2006. Non-pollinator agents of selection on floral traits. Ecology and evolution of flowers, pp.120-138.

  \newpage


\bibliographystyle{newphy} 
\renewcommand*{\bibfont}{\raggedright}
\bibliography{manual}

\end{document}